solutions/client-setup/setters.yaml

```diff
@@ -74,17 +91,23 @@ data:
   # the git repo URL, for example
   # https://github.com/GITHUB-ORG/REPO-NAME
   # https://AZDO-ORG@dev.azure.com/AZDO-ORG/AZDO-PROJECT/_git/REPO-NAME
```

Comment:
For further clarity should we add a description of is expected in the git repo, core landing zone configs, client lz, etc?

Reply:
can't hurt, added more details a few lines above